packages/components/src/local/adjudicate-turn-form/types/props.d.ts

```diff
@@ -1,19 +1,33 @@
-export default interface PropTypes {
-  /**
-   * The header text for the form
-   */
-  formHeader: string
-  /**
-   * All types in this definition are options for a form input
-   */
-  formData: {
+interface FormData {
+  populate: {
     status: Array<string>
     speed: Array<number>
     visibleTo: Array<{
       name: string
       colour: string
-      selected: boolean
     }>
     condition: Array<string>
   }
+  values: {
+    plannedRouteStatusVal: string
+    statusVal: string
+    speedVal: number
+    visibleToVal: string
+    conditionVal: string
+  }
+}
+
+export default interface PropTypes {
+  /**
+   * The header text for the form
+   */
+  formHeader: string
+  /**
+   * All types in this definition are options for a form input
+   */
+  formData: FormData
+ /**
+   * The method for posting data back to state
+   */
+  postBack?: any
```

Comment:
I think it's worth capturing the structure of the _data nugget_ that gets posted back, since by definition it's a data structure that's shared between multiple components.

_Maybe_ the TS type will be something like the types we encountered in MappingContext:
```
  setSelectedAsset: React.Dispatch<React.SetStateAction<SelectedAsset>>
```

Reply:
Basically, any time you see `any` here it's because I was trying to focus on getting the functionality working rather than getting bogged down in typing. However it's a fair cop, I shouldn't really be getting into the habit of raising PR's without going through the `any` entries. I'll make sure this and others are as resolved to types as possible before I mark this as resolved.